src/external/VTK.jl

```diff
@@ -0,0 +1,49 @@
+# Uses WriteVTK.jl to convert scfres structure to VTk file format
+import WriteVTK: vtk_grid, vtk_save
+
+"""
+    save_scfres(VTKFileName, scfres)
+
+The function takes in the VTK filename and the scfres structure and stores into a VTK file.
+Grid Values:
+ρ_real -> real values of scfers.ρ
+ψreal_i_j -> Real values of Bloch waves in real space, where i and j are Kpoint and EigenVector indexes respectively
+ρspin -> Real of ρspin are stored if ρspin in present
+MetaData:
+Energy and occupation.
+"""
+function save_scfres(VTKFileName::AbstractString, scfres)
```

Comment:
Also: We use snail case for variable names, so `vtk_file_name` (or just `filename`).